exercises/concept/lasagna/.meta/exemplar.jl

```diff
@@ -1,3 +1,7 @@
+expected_bake_time = 60
```

Comment:
Shouldn't this be a constant?

Reply:
Totally, and at one stage it was. Some sort of odd reversion...

Reply:
Definitely there now.